Tasks/PowerShellV2/task.json

```diff
@@ -85,7 +85,7 @@
             "type": "pickList",
             "label": "ErrorActionPreference",
             "required": false,
-            "defaultValue": "stop",
+            "defaultValue": "default",
```

Comment:
Could it be a breaking change for a customers who specify errorActionPreference as 'Stop' in the pipeline, but it is not 'Stop' on agent environment?